elasticsearch/templates/statefulset.yaml

```diff
@@ -128,7 +128,11 @@ spec:
         {{- end }}
 {{ end }}
       {{- if .Values.extraVolumes }}
```

Comment:
Could you add a comment similar to the one added in Filebeat PR (https://github.com/elastic/helm-charts/blob/master/filebeat/templates/daemonset.yaml#L93-L98) so we know that this is done for backward compatibility?